tidb-configuration-file.md

```diff
@@ -88,12 +88,16 @@ The TiDB configuration file supports more options than command-line parameters.
 - Determines whether to treat the `utf8` character set in old tables as `utf8mb4`.
 - Default value: `true`
 
-### `alter-primary-key`
+### `alter-primary-key` (Deprecated)
 
 - Determines whether to add or remove the primary key constraint to or from a column.
 - Default value: `false`
 - With this default setting, adding or removing the primary key constraint is not supported. You can enable this feature by setting `alter-primary-key` to `true`. However, if a table already exists before the switch is on, and the data type of its primary key column is an integer, dropping the primary key from the column is not possible even if you set this configuration item to `true`.
 
+> **Note:**
+>
+> This configuration item no longer takes effect. If you need to add or delete the primary key, use `NONCLUSTERED` instead. For more details about the primary key of the `CLUSTERED` type, refer to [clustered index](/clustered-indexes.md).
```

Comment:
Please align https://github.com/pingcap/docs-cn/pull/5834/files#diff-0c6e827c2bde2f920447a23365ce5f4603fe7067d99ec1d1dfc3d22bb64c51e3R101 this line, thanks!

Reply:
@TomShawn PTAL again thx!